public/app/features/provisioning/api/endpoints.ts

```diff
@@ -477,7 +477,7 @@ export type SyncStatus = {
   message?: string[];
   scheduled?: number;
   started?: number;
-  state: 'error' | 'pending' | 'success' | 'working';
+  state?: 'error' | 'pending' | 'success' | 'working' | '';
```

Comment:
If you added these types manually, they'll be overwritten next time the client is generated. Should probably rename this file to `endpoints.gen.ts` so clearer show that it's generated.